Fix typecast in Metadata::from implementation.

sullivanmj opened the pull request:
Fixes #37.

robin-nitrokey commented:
Thanks for the PR!  Does this fix the Windows build for you or are there any other issues?  And can you please add a changelog entry?

sullivanmj commented:
My pleasure :) The crate builds on Windows for me with the suggested fix. I added a changelog entry and removed the import from my previous commit. Please let me know if there is anything else I need to do.

robin-nitrokey reviewed (approved):
Thanks, looks good!
